rbi/core/io.rbi

```diff
@@ -1739,7 +1739,7 @@ class IO < Object
   # ```
   sig do
     params(
-        arg0: String,
+        arg0: BasicObject,
```

Comment:
I think we would need `Object` and not `BasicObject` since `to_s` is first defined on `Object`. Same with the other usages.

Reply:
Good catch, I changed the types to `Object`.